composer.json

```diff
@@ -2,7 +2,7 @@
     "name": "lizardmedia/module-product-attachments",
     "description": "Products attachment by LizardMedia",
     "require": {
-        "php": "~7.1.0|~7.2.0|~7.3.0",
+        "php": "~7.1",
```

Comment:
please leave all versions, because it is more expressive and add 7.4 version after next pipe